File: omod/src/main/webapp/resources/scripts/services/attachmentService.js
angular.module('att.service.attachmentService').factory('Attachment', function($resource) {
    return $resource("/"   OPENMRS_CONTEXT_PATH   "/ws/rest/v1/attachment/:uuid", {
        uuid : '@uuid'
    }, {
        query : {
            method : 'GET',
            isArray : false
        }
    // OpenMRS RESTWS returns { "results": [] }
    });
})factory('AttachmentService', function(Attachment) {
return {
         /**
          * @param params to search for attachments
          */
         getAttachments : function(params) {
             return Attachment.query(params).$promise.then(function(res) {
                 return res.results;
             });
         }
     }
 });

File: omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentService.java
package org.openmrs.module.attachments.rest;

import org.openmrs.Encounter;
import org.openmrs.Patient;
import org.openmrs.Visit;
import org.openmrs.module.attachments.obs.Attachment;

import java.util.List;

/**
 * Created by dileka on 3/17/18.
 */
public interface AttachmentService {
	
	List<Attachment> getAttachments(Patient patient, Visit visit, Encounter encounter, boolean includeRetired);
	
}
